docs/contributing/dev-setup/runtests.rst

```diff
@@ -0,0 +1,16 @@
+.. _runtests:
+
+Run the Tests
+=============
+
+To run the tests provided by a specific plugin, start the django development server in the development vm with `pulp-manager runserver`.
+You need to create a *pulp-smash* config file with `pulp-smash config create` (this is ony needed once).
```

Comment:
Pulp-Smash is used to write functional tests. Perhaps, adding a link to the documentation. [Config](https://pulp-smash.readthedocs.io/en/latest/configuration.html#)

Besides the user can choose different test runners in order to run functional tests, for instance.
```bash
py.test pulpcore/tests/functional/
```

or 

```bash
python -m unittest pulpcore/tests/functional/
```